--- ./config.m4~	2005-09-12 01:14:30.000000000 +0300
+++ ./config.m4	2005-09-12 01:15:54.000000000 +0300
@@ -11,12 +11,12 @@
 
   PHP_PREFIX=`$PHP_CONFIG --prefix`
   AC_MSG_CHECKING(for PHP executable in $PHP_PREFIX/bin)
-  if test -x $PHP_PREFIX/bin/php; then
+  if test -x $PHP_PREFIX/bin/php4; then
     PHP_VERSION=`$PHP_CONFIG --version`
 	AC_MSG_RESULT(found version $PHP_VERSION)
 	case $PHP_VERSION in
        4.3*|4.4*[)]
-	   PHP=$PHP_PREFIX/bin/php
+	   PHP=$PHP_PREFIX/bin/php4
 	;;
 	   *[)]
 	   AC_MSG_ERROR(
